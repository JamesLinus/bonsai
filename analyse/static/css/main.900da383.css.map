{"version":3,"sources":[],"names":[],"mappings":"","file":"static/css/main.900da383.css","sourceRoot":""}