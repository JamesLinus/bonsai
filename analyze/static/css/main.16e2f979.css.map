{"version":3,"sources":[],"names":[],"mappings":"","file":"static/css/main.16e2f979.css","sourceRoot":""}