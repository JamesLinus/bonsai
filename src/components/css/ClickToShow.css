.ClickToShow {
  position: relative;
}

.ClickToShow-Panel {
  position: absolute;
  top: 1em;
  left: 0;
  background: white;
  border: 1px solid black;
  padding: 5px;
  z-index: 1;
}
