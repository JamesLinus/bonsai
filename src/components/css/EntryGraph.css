.EntryGraph {
  text-align: left;
}
.EntryGraph,
.EntryGraph ul {
  margin-left: 1em;
}
