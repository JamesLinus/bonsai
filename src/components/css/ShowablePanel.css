.ShowablePanel {
  position: relative;
}

.ShowablePanel-Panel {
  background: white;
  border: 1px solid black;
  box-shadow: black 1px 1px 10px;
  padding: 5px;
  position: absolute;
  top: 1em;
  z-index: 1;
}

.ShowablePanel-Panel--left {
    left: 0;
}
.ShowablePanel-Panel--right {
    right: 0;
}
