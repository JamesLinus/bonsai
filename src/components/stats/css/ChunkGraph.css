.ChunkGraph {
  text-align: left;
}
.ChunkGraph,
.ChunkGraph ul {
  margin-left: 1em;
}
