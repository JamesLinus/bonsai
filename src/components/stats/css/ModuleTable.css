.ModuleTable {
  border-collapse: collapse;
}

.ModuleTable thead th,
.ModuleTable thead td {
  padding: 10px 5px;
  border-bottom: 1px solid #ccc;
}

.ModuleTable tr:hover {
  background: #ddd;
}

.ModuleTable tr:target {
    background: #fdf5d4;
}

.ModuleTable td {
  padding: 4px 2px;
  border-bottom: 1px solid #eee;
}
