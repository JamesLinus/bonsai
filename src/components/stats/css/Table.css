.Table {
  border-collapse: collapse;
}

.Table thead th,
.Table thead td {
  padding: 10px 5px;
  border-bottom: 1px solid #ccc;
}

.Table tr:hover {
  background: #ddd;
}

.Table tr:target {
  background: #fdf5d4;
}

.Table td {
  padding: 4px 2px;
  border-bottom: 1px solid #eee;
}

.Table td.filter {
  text-align: center;
}

.Table td.number {
  text-align: right;
}
