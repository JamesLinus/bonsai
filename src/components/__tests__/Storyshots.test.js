import initStoryshots from 'storyshots';

import 'bootstrap/dist/css/bootstrap.min.css';
import '../../index.css';

initStoryshots();
