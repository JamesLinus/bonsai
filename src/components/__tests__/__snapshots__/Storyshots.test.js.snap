exports[`Storyshots App Default View 1`] = `
<div
  className="App">
  <nav
    className="navbar navbar-default navbar-fixed-top navbar-inverse">
    <div
      className="container-fluid">
      <div
        className="navbar-header">
        <a
          className="navbar-brand"
          href="/">
          Webpack Dependency Size
        </a>
      </div>
    </div>
  </nav>
  <aside
    className="container-fluid">
    <div
      className="row">
      <div
        className="col-xs-12">
        <div
          onClick={[Function]}
          style={undefined}>
          <input
            className="form-control"
            id="drag-drop-upload"
            style={
              Object {
                "display": "none",
              }
            }
            type="file" />
          <div
            className="well well-sm clearfix">
            <div
              className="form-horizontal">
              <div
                className="form-group col-sm-6">
                <label
                  className="control-label">
                  Drag & Drop your 
                  <code>
                    stats.json
                  </code>
                   file here
                </label>
                <span
                  className="col-sm-12 help-block"
                  id="drag-drop-helpblock">
                  Run 
                  <kbd>
                    webpack --json > stats.json
                  </kbd>
                   to get started.
                </span>
              </div>
              <div
                className="form-group col-sm-6"
                onClick={[Function]}>
                <label
                  className="col-sm-2 control-label"
                  htmlFor="data-file-picker">
                  File
                </label>
                <div
                  className="col-sm-10">
                  <select
                    className="form-control"
                    id="data-file-picker"
                    onChange={[Function]}>
                    <option
                      value="" />
                  </select>
                </div>
                <span
                  className="col-sm-12 help-block"
                  id="drag-drop-helpblock">
                  or pick an existing stats file.
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </aside>
  <main
    className="container-fluid">
    <div
      className="row">
      <div
        className="col-xs-12" />
    </div>
  </main>
</div>
`;
